mantidproject/mantid
# Elemental analysis and release notes

We have removed the original version of elemental analysis (it has never been in a release) at the request of the user.


**To test:**
Ask me for some data

Under Muon's check that there is only one elemental analysis
Open elemental analysis
Load the data - need to press the load button
When you click one of the detectors a plot will appear
Click some elements and lines should appear on the plot



Fixes #25080. 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
